daweb/js1/pole-data/cvlekce/pole-v-divadle.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Pole v divadle
+demand: 1
+lead: Vyzkoušejte si vytvořit jednoduchá pole.
+solutionAccess: lock
+---
+
+Cvičení provádějte ve vašem JavaScript souboru.
+
+1. Vytvořte nějaké pole celých čísel, například počty diváků na několika po sobě jdoucích divadelních představeních. Použijte metodu `push` a přidejte do pole počet diváků na novém představení.
+1. Vytvořte nějaké pole desetinných čísel, například zaplněnost divadla v několika po sobě jdoucích představeních. Poté pomocí metody `pop` odstraňte poslední hodnotu v poli.
+1. Vytvořte nějaké pole řetězců, například seznam názvů několika divadelních představení, která divadlo hraje. Uložte toto pole do proměnné `plays`. Uložte do nějaké proměnné druhou položku tohoto pole.
+1. Uložte do proměnné `reviews` pole hodnocení, které obdržela divadelní hra _Plyšáci na útěku_ v různých recenzních časopisech. Hodnocení je vždy objekt obsahující název recenzního časopisu jako řetězec a samotné hodnocení jako číslo mezi 1 až 10.
+
```

Comment:
Jak je to prostrkané slovíčkem nějaké, tak se obávám, že pro účastnice bude hodně těžké poznat, jestli dělají to, co se po nich chce a jestli to dělají správně.